sdcm/cluster.py

```diff
@@ -2493,10 +2505,14 @@ def install_scylla_monitoring_prereqs(self, node):
             """)
         else:
             prereqs_script = dedent("""
+                apt-get update
```

Comment:
As discussed `else` should raise exception and this script is for Debian

Reply:
done